### ISSUE TITLE
related-resources-manager.htm template has absolute path and causing 500 error

### ISSUE THREAD

**Issue by EnisBerk** [archesproject/arches#7503]
**Describe the bug**

related-resources-manager.htm template is not working and causing 500 error. Problem is that an image file's path is hard coded and starting with a slash '/'.

**To Reproduce**

0. Requires localization with more than 1 language
1. Go to localhost.com:8000/en/search?paging-filter=1&tiles=true
2. Set STATIC_URL to something other than '/'
3. Check console and server logs for error

**Screenshots**

```
[07/Jun/2021 07:27:56] "GET /en/search/resources HTTP/1.1" 200 4839
Attempted access to '/img/Workflow.png' denied.
Internal Server Error: /en/templates/views/resource/related-resources/related-resources-manager.htm
Traceback (most recent call last):
  File "/root/venvs/archesenv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/root/venvs/archesenv/lib/python3.8/site-packages/django/core/handlers/base.py", line 124, in _get_response
    raise ValueError(
ValueError: The view arches.app.views.main.templates didn't return an HttpResponse object. It returned None instead.
[07/Jun/2021 07:27:57] "GET /en/templates/views/resource/related-resources/related-resources-manager.htm?_dc=5.1.4 HTTP/1.1" 500 79506
```
**Expected behavior**

Removing slash from the path fixes the problem.
[line1](https://github.com/archesproject/arches/blob/3a8f8764a5f4a9f340a986c59a31b083354e0158/arches/app/templates/views/resource/related-resources/related-resources-manager.htm#L9)
[line2 ](https://github.com/archesproject/arches/blob/3a8f8764a5f4a9f340a986c59a31b083354e0158/arches/app/templates/views/resource/related-resources/related-resources-manager.htm#L311)

**Your Arches Information**

*   Version used: 5.1.4 (also checked 5.2.x)
*   Browser Name and version: Firefox

**Comment by EnisBerk** [comment]
Related to https://github.com/archesproject/arches/issues/4278

**Comment by chiatt** [comment]
Hey @EnisBerk, thanks for submitting this. @robgaston has submitted a fix in `stable/5.2.x`. We were not able to reproduce the issue, but the change was easy to make, so hopefully it clears up the error for you.

**Comment by EnisBerk** [comment]
Hi @chiatt and @robgaston, I just tested this fix with stable/5.2.x and I realized @robgaston missed a [line](https://github.com/archesproject/arches/blob/6088a392abef1ad9cb818ac1cfab567a4d755087/arches/app/templates/views/resource/editor.htm#L158)  which has "/img/inspection.png"

To reproduce, I am using s3 to serve files with boto3 and two languages are active. If any of these matters.

**Comment by chiatt** [comment]
This should now be resolved by #8322
